ncs/sign_script.py

```diff
@@ -14,44 +14,24 @@
 """
 from __future__ import annotations
 
-import math
-
 import cbor2
-import uuid
+import importlib.util
+import sys
 
-from argparse import ArgumentParser
 from pathlib import Path
-from cryptography.hazmat.primitives import hashes
-from cryptography.hazmat.primitives.serialization import load_pem_private_key, load_der_private_key
-from cryptography.hazmat.primitives.asymmetric.utils import decode_dss_signature
-from cryptography.hazmat.primitives.asymmetric import ec
-from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePrivateKey
-from cryptography.hazmat.primitives.asymmetric.ed25519 import Ed25519PrivateKey
-from cryptography.hazmat.primitives.asymmetric.ed448 import Ed448PrivateKey
-from collections import defaultdict
 from enum import Enum, unique
-
-#
-# User note:
-#   Rename the files to 'key_private_<KEY>.der' if you are using keys in DER format.
-#
-PRIVATE_KEYS = {
-    0x40000000: Path(__file__).parent / "key_private.pem",
-    0x4000AA00: Path(__file__).parent / "key_private_OEM_ROOT_GEN1.pem",
-    0x40022100: Path(__file__).parent / "key_private_APPLICATION_GEN1.pem",
-    0x40032100: Path(__file__).parent / "key_private_RADIO_GEN1.pem",
-}
+from suit_generator.suit_kms_base import SuitKMSBase
+from suit_generator.suit_sign_script_base import (
+    SuitEnvelopeSignerBase,
+    SignatureAlreadyPresentActions,
+    SuitSignAlgorithms,
+)
 
 
 @unique
 class SuitAlgorithms(Enum):
```

Comment:
Since you've already specified `SuitEncryptAlgorithms`, consider changing to `SuitSignAlgorithms`

Reply:
`SuitSignAlgorithms` already has a different function (see it is imported above), however to be consistend changed the above to SuitCoseEncryptAlgorithms and SuitCoseSignAlgorithms